collect_app/build.gradle

```diff
@@ -63,7 +63,7 @@ android {
     defaultConfig {
         applicationId("org.odk.collect.android")
         minSdkVersion(16)
-        targetSdkVersion(22)
+        targetSdkVersion(25)
```

Comment:
Updating the targetSdkVersion was not the concern of the issue. Please try not to make changes more than the scope of the issue